content/en/blog/2020/tradewinds-2020/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Istio in 2020: Following the Trade Winds
```

Comment:
I'm suspicious of the extra : in the title of the metadata.

Reply:
and...you were correct! Thanks.